### PR TITLE
Updated rolling-math documentation and added license files.

### PR DESCRIPTION
**Pull request by arbitrary-kdawg**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+rolling-math npm Package
+
+Copyright (C) 2021 Arbitrary Execution
```
Comment (arbitrary-echo) [inline LICENSE]: Should 2022 be here as well?
```suggestion
Copyright (C) 2021-2022 Arbitrary Execution
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+rollingMath = new RollingMath(5);
+```
+
+The window size (`5` in this case) represents the maximum number of previous elements the RollingMath
+object will store.  Each time a new value is added, the oldest value is removed once the window
+size limit has been reached.  Window size corresponds directly with time--if you are sampling once
+per day, and the window size is 5, you have a 5-day Simple Moving Average (SMA) filter.
+
+
+### Functions
```
Comment (arbitrary-echo) [inline README.md]:
```suggestion
### Methods
```

**Review by arbitrary-echo — APPROVED** [review]
Minor suggestions.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "rolling-math",
   "version": "0.0.1",
```
Comment (y0wl) [inline package.json]:
```suggestion
  "version": "0.0.2",
```

**Review by y0wl — APPROVED** [review]

**Pull request merged**